microsite/src/main/tut/datatypes/schedule.md

```diff
@@ -78,7 +79,7 @@ val jitteredExp = Schedule.exponential(10.milliseconds).jittered
 Modifies the delay of a schedule:
 
 ```scala mdoc:silent
-val boosted = Schedule.spaced(1.second).delayed(_ + 100.milliseconds)
+val boosted = Schedule.spaced(1.second).delayed(_ + 100.milliseconds.relative)
```

Comment:
All these `.relative` make me want the "syntax" on `Delay`, e.g. `100.millis` could give you a `Delay`. If we don't get something like that, then this will be a source breaking change for existing users. Let me think about that for a bit.